openstack/cronus/sub-charts/maillog/templates/deployment.yaml

```diff
@@ -2,43 +2,26 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: maillog
-  {{- if .Values.annotations }}
-  annotations:
-    {{ toYaml .Values.annotations | indent 4 }}
-  {{- end  }}
-  {{- if .Values.labels }}
-  labels:
-    {{ toYaml .Values.labels | indent 4 }}
-  {{- end }}
 spec:
-  replicas: {{ default .Values.deployment.replicas }}
+  replicas: {{ .Values.deployment.replicas }}
   strategy:
-{{- if eq "RollingUpdate" .Values.updateStrategy }}
```

Comment:
I don't think we would change this. And if we did, then I guess the chart would just be blank here, which then actually just does a rolling update in the first place :)